zanata-war/src/main/webapp/WEB-INF/layout/project/settings-tab.xhtml

```diff
@@ -74,6 +74,8 @@
   <a4j:param name="val2" assignTo="#{isRestricted}"/>
 </a4j:jsFunction>
 
+<a4j:jsFunction name="runDelete" action="#{projectHome.updateStatus('O')}" />
```

Comment:
I know this is an existing implementation, but it really have big security hole as user can modify any char in the javascript. I would prefer something like `projectHome.deleteProject` and let the backend handle the char.